ccstruct/matrix.h

```diff
@@ -38,7 +38,7 @@
 class BLOB_CHOICE;
 class BLOB_CHOICE_LIST;
 
-#define NOT_CLASSIFIED reinterpret_cast<BLOB_CHOICE_LIST*>(0)
+#define NOT_CLASSIFIED static_cast<BLOB_CHOICE_LIST*>(0)
```

Comment:
At first sight it could probably be 0, NULL, or nullptr, with or without the cast, i.e., one of 6 possibilities. I would definitely keep the cast to let the compiler check for appropriate usage, but otherwise I'm neutral.